grpc/handlers.go

```diff
@@ -98,6 +98,11 @@ func (srv *grpcServer) ListDirectory(
 			})
 		}
 
+		var backupCount = 0
+		if e.BackedUp {
+			backupCount = 1
```

Comment:
@jsonsivar 

Is that a correct counter?

Reply:
Yeah for now it is, we are storing it as a bool internally but didn't change that yet because we can do that with the broader piece of work that's upcoming related to that (online/offline state management and multi-hub support).